CHANGELOG.rst

```diff
@@ -25,7 +30,6 @@ Unreleased
 
 * Added API endpoint for fetching catalogs and catalog courses.
 
-
```

Comment:
I think that might have been an artefact of a rebase, because it's now not showing CHANGELOG.rst in my diff.

But I'll update this file when it comes time to merge.